src/settings.ts

```diff
@@ -149,6 +149,8 @@ export const PLAYA_VENUE_STYLES: Record<string, CSSProperties> = {
   },
 };
 
+export const POSTER_LOGO_SIZE = 100;
```

Comment:
Any reason to have it like this rather than in classes

Reply:
Not really, I'm just using it with img's width prop like this `<img width={POSTER_LOGO_SIZE}/>`

Reply:
Can we do it with the classes just to follow the patterns?